pyproject.toml

```diff
@@ -41,8 +40,8 @@ dependencies = [
 	"keyring >= 15.1; platform_machine != 'ppc64le' and platform_machine != 's390x'",
 	"rfc3986 >= 1.4.0",
 	"rich >= 12.0.0",
-	"packaging",
-	"id",
+	"packaging >= 24.0",
```

Comment:
Please correct me if I'm wrong but support for PEP 639 (as mentioned in the changelog entry) was only added in 24.2. See release notes: https://github.com/pypa/packaging/releases/tag/24.2

Shouldn't this be limited to `>= 24.2`?

Reply:
It is correct that support for metadata 2.4 ha been added to `packaging` only in version 24.2. However, using `packaging` to parse metadata only requires functionality added in version 24.0. With `packaging` version < 24.2 >= 24.0 everything added here still works, thus I didn't feel like requiring the latest version.

Reply:
Makes sense, but I think the changelog should reflect this. From reading the Discourse thread this is eagerly awaited. Making it so that `pip install -U twine` doesn't actually include support for `2.4` but that `pip install -U packaging` is also needed will probably confuse people.